packages/documentation/src/pages/getting-started/index.mdx

```diff
@@ -97,14 +108,46 @@ These instructions cover building and running VA.gov locally, including configur
    $ yarn fetch-drupal-cache
    ```
 
-9. Start the local development server.
+9a. Start the local development server.
+   ```bash
+   $ yarn watch
+   ```
+
+   The watch is complete when the CLI says `Compiled successfully`.
+
+10a. Open [http://localhost:3001](http://localhost:3001) in a browser
+
+### content-build
+
+In order to run `content-build`, you must first generate the CSS and JS files within `vets-website` 
+(follow steps above). If you don't you will see a build error in the console instructing you to first
+generate files in `vets-website`.
+
+If you don't need to locally view static pages and are just working on applications, you don't need to set this part up.
+
+7b. Navigate to the `content-build` repository, then install dependencies via `yarn`
+   ```bash
+   $ cd content-build
+   $ yarn install
+   ```
+
+8b. Build `content-build`. Make sure you [configured the SOCKS proxy](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#configure-the-socks-proxy) to fetch content from Drupal.
+   ```bash
+   $ yarn build
+   ```
+
+   If you do not have access to the SOCKS proxy, you can alternately fetch the latest cached version of the content.
+   ```bash
+   $ yarn fetch-drupal-cache
+
```

Comment:
Github won't let me make this suggestion inside a suggestion block, but we're missing triple backticks (```) here to close off the code snippet above.

Reply:
updated locally